### DISCUSSION TITLE
fetching array items by their index position

### DISCUSSION THREAD

**Discussion by acivitillo** [lowdefy/lowdefy#863] [Docs]
I just found out that fetching items from an array is done using the `.` notation instead of the `[]` one. I can send you a PR to update the docs, but the question is where should we document this in the docs? My proposal is to add a tutorial "Working with Objects".

Say we have a json response like this one:

```json
[
  {"label":"Italy","value":1,"disabled":false,"filterString":null}
  , {"label":"Germany","value":1,"disabled":false,"filterString":null}
]
```

And a request defined like so:

```yaml
connections:
  - id: app_api
    type: AxiosHttp
    properties:
      baseURL:  http://127.0.0.1:8000
requests:
  - id: get_countries
    type: AxiosHttp
    connectionId: app_api
    properties:
      url: /dropdowns/dd_countries
```

And we wanted to show the country label of the `first` item of the countries array, we would do it like this:

```yaml
id: content
    type: Result
     properties:
          title: Welcome to your Lowdefy app
           subTitle: 
                _request: get_countries.data.0.label #<- check here
```

Basically we need to do `get_countries.data.0.label` instead of `get_countries.data[0].label`. We also need to use `.data` which is documented here in an answer but it is not super clear from the docs.

**Comment by Gervwyk** [comment]
Hi @acivitillo 

Thanks for the detailed feedback! I agree more detailed explanations and examples on this will really help.
We've started a "How to" series of articles covering core concepts and examples. A "How to work with data in Lowdefy" article that covers short examples like this will be a nice addition. I'll add this.

We will be posting the first 2 "How to" episodes in the series within the next two weeks.
If you have any feedback, ideas or feel like contributing, please reach out 💫

**Comment by acivitillo** [comment]
Great, I'm working on a demo app here with my team: https://github.com/dyvenia/timesheets

So as things come up we can provide feedback/documentation. 

I'll be waiting for your "How To" series then.

**Reply by Gervwyk** [reply]
Nice! Looks like a great problem to solve using Lowdefy. 

We have also decided to create a time tracking application that integrates with GitHub projects and issues. We will be creating an API that logs webhook data posted from GitHub to MongoDB, then a Lowdefy front end to fix time "errors" and create nice reports based on sprint planning and estimations. 

The idea is that we'll calculate the duration of an issue in the "working on it" column on our project boards, then build reports on that data and issue labels. 

This will be a very generic application so we'll also open-source it as an example / community project. Will update here once the first version is live.
